addons/test_mail/tests/test_ir_actions.py

```diff
@@ -3,19 +3,53 @@
 
 from odoo.addons.base.tests.test_ir_actions import TestServerActionsBase
 from odoo.addons.test_mail.tests.common import TestMailCommon
+from odoo.tests import tagged
+from odoo.tools import mute_logger
 
 
+@tagged('ir_actions')
 class TestServerActionsEmail(TestMailCommon, TestServerActionsBase):
 
+    def setUp(self):
+        super(TestServerActionsEmail, self).setUp()
+        self.template = self._create_template(
+            'res.partner',
+            {'email_from': '{{ object.user_id.email_formatted or object.company_id.email_formatted or user.email_formatted }}',
+             'partner_to': '%s' % self.test_partner.id,
+            }
+        )
+
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
     def test_action_email(self):
```

Comment:
"test_action_mail_post" ?

EDIT: well actually this still sends an email, was trying to match the action "state" field, but it can stay.